public/app/features/commandPalette/actions/staticActions.ts

```diff
@@ -31,15 +25,15 @@ function navTreeToActions(navTree: NavModelItem[], parent?: NavModelItem): Actio
       continue;
     }
 
-    const action: Action = {
+    const section = isCreateAction ? t('command-palette.section.actions', 'actions') : getPagesSectionTranslation();
```

Comment:
think the default here should be `Actions`, also do you need to run i18n-extract as well?

Reply:
😭